kubernetes-ingress/http.md

```diff
@@ -8,7 +8,20 @@ Deploy the demo app:
 kubectl apply -f https://raw.githubusercontent.com/istio/istio/release-1.11/samples/bookinfo/platform/kube/bookinfo.yaml
 ```
 
-*Note: this is just deploying the demo app, it's not adding any Istio components*
+Note that with Cilium Service Mesh there is no Envoy sidecar created alongside each of the demo app microservices. 
```

Comment:
```suggestion
*Note: this is just deploying the demo app, it's not adding any Istio components*

You can confirm that with Cilium Service Mesh there is no Envoy sidecar created alongside each of the demo app microservices. 
```

I think it's worth leaving in the note that this is just deploying the sample app, but nothing else, from Istio (given that the YAML comes from the Istio repo, so people might wonder!)